Source/Core/DolphinQt/Config/ToolTipControls/ToolTipWidget.h

```diff
@@ -49,6 +51,7 @@ class ToolTipWidget : public Derived
 
   virtual QPoint GetToolTipPosition() const = 0;
 
+protected:
```

Comment:
These variables shouldn't be protected.  It was rather intentional that they were private because the private functions (or public ones) are the only thing that should manage these variables.

Reply:
Yeah, that's a mistake, I had assumed they were public.